packages/cli/cli/src/cli.ts

```diff
@@ -187,16 +188,31 @@ function addInitCommand(cli: Argv<GlobalCliOptions>, cliContext: CliContext) {
                 .option("openapi", {
                     type: "string",
                     description: "Filepath or url to an existing OpenAPI spec",
+                })
+                .option("url", {
+                    type: "string",
+                    description: "The url for the docs website (e.g. --url https://buildwithfern.com/docs)",
                 }),
         async (argv) => {
             if (argv.api != null && argv.docs != null) {
                 return cliContext.failWithoutThrowing("Cannot specify both --api and --docs. Please choose one.");
             } else if (argv.docs != null) {
+                let docsUrl: ReadMeDocsUrl;
+                if (argv.url != null) {
+                    docsUrl = new ReadMeDocsUrl(argv.url);
+                    if (!docsUrl.isUrlValid()) {
+                        return cliContext.failWithoutThrowing("--url must be a valid url");
+                    }
+                    if (argv.organization == null) {
+                        return cliContext.failWithoutThrowing("--organization must be specified when using --url");
```

Comment:
nit: would be sick if it just prompoted user for organization https://stackoverflow.com/questions/33858763/console-input-in-typescript

Reply:
for sure, let me cut a ticket for this (will do in a new pr)

Reply:
https://linear.app/fern-api/issue/FER-547/prompt-user-to-input-an-organization-when-running-init-docs-url